docs/api/python/autograd/autograd.md

```diff
@@ -76,7 +82,63 @@ Detailed tutorials are available in Part 1 of
 [the MXNet gluon book](http://gluon.mxnet.io/).
 
 
+# Higher order gradient
+
+Some operators support higher order gradients. Meaning that you calculate the gradient of the
```

Comment:
I think "Meaning that you calculate the gradient of the gradient" is redundant. People who use this package should understand what second order gradient is. Technically speaking it is the second order gradients of a loss function with respect to the input variables.

Reply:
ok